app-text/ghostwriter/patches/ghostwriter-1.5.0.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/src/AppSettings.cpp b/src/AppSettings.cpp
+index 1bc0035..5e1ae36 100644
+--- a/src/AppSettings.cpp
++++ b/src/AppSettings.cpp
+@@ -483,7 +483,11 @@ AppSettings::AppSettings()
+         userDir = QFileInfo(settings.fileName()).dir().absolutePath();
+ 
+         QStringList translationPaths;
++#ifdef Q_OS_HAIKU
++        translationPaths.append((appDir + "/../data/ghostwriter/translations"));
```

Comment:
This might not work, and it would be better to change it differently.
Using find_directory with B_SYSTEM_TRANSLATORS_DIRECTORY might be an option, but I did not investigate.

You could, alternatively, use sed in a PATCH function to patch this line:
```
         translationPaths.append(appDir + "/translations");
```
with
```
PATCH()
{
	sed -i \
		-e "/translationPaths.append/ \
			s|appDir + \"/translations\"|\"$dataDir/ghostwriter/translations\"|;" \
		src/AppSettings.cpp
}
```